charts/argo-cd/templates/argocd-application-controller/servicemonitor.yaml

```diff
@@ -14,6 +14,10 @@ metadata:
     {{- with .Values.controller.metrics.serviceMonitor.additionalLabels }}
       {{- toYaml . | nindent 4 }}
     {{- end }}
+  annotations:
```

Comment:
@mkilchhofer Does this really work when `annotations: {}` because rendered block will be:
```yaml
annotations:
spec:
  endpoints: <etc>
```

Reply:
Ahh good point. On normal (non-CRD) objects it should not be a problem.

Reply:
I fixed with a better solution, it won’t render the annotation line if there aren’t any annotations